src/selic-entries.ts

```diff
@@ -0,0 +1,11 @@
+export async function selicEntries(initialDate: string, finalDate: string) {
+    const url = new URL('https://api.bcb.gov.br/dados/serie/bcdata.sgs.11/dados?formato=json');
+    const params = new URLSearchParams;
+    params.append('dataInicial', initialDate);
+    params.append('dataFinal', finalDate);
+    url.search = params.toString();
+    
+    const response = await fetch(url.toString());
+    const selic_entries = await response.json();
+    return JSON.stringify(selic_entries);
```

Comment:
Why are you converting a data object into a string? One should be able to consume the outcome as an object

```json
[
  {
    "data":"08/03/2024",
    "valor":"0.041957"
  }
]
````

and not as a string

```typescript
'[{"data":"08/03/2024","valor":"0.041957"}]'
```